README.md

```diff
@@ -62,7 +62,7 @@ export default function App() {
         y: cy.value,
       }
     },
-    onActive: ({ translationX, translationY }, context) => {
```

Comment:
This was also incorrect and this example didn't work. The `context` variable is declared above in the component, so it overrode the variable with `undefined`.